javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/declarations/JavaParserInterfaceDeclaration.java

```diff
@@ -186,7 +186,7 @@ public boolean isTypeParameter() {
             fields.add(new ResolvedFieldDeclaration() {
                 
                 @Override
-                public AccessSpecifier accessSpecifier() {
+                public Modifier.Keyword accessSpecifier() {
```

Comment:
Mmm, AccessSpecifier includes the default package access for which there is no corresponding modifier

Reply:
I've been trying to refactor AccessSpecifier away because it overlapped with modifiers (and I think there's even a third one) - but if it is too much it can be reverted.